src/rules/flake8_use_pathlib/violations.rs

```diff
@@ -0,0 +1,257 @@
+use ruff_macros::derive_message_formats;
+
+use crate::define_violation;
+use crate::violation::Violation;
+
+// PTH100
+define_violation!(
+    pub struct PathlibAbspath;
+);
+impl Violation for PathlibAbspath {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("`os.path.abspath` should be replaced by `.resolve()`")
+    }
+}
+
+// PTH101
+define_violation!(
+    pub struct PathlibChmod;
+);
+impl Violation for PathlibChmod {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("`os.chmod` should be replaced by `.chmod`")
```

Comment:
Should this be `.chmod()` at the end, with parens?

Reply:
(Genuinely asking, since I know some of these are actually properties, like `.parent`.)

Reply:
Yes, was a mistake